diagnostic.md

```diff
@@ -94,4 +94,6 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+hobby='excercise'
+name=[given]+[surname']
```

Comment:
Not quite. teamMember['name'] would actually return the object that is store under the key of 'name': {given: 'Jason', surname: 'Weeks'}